test/test_ops.py

```diff
@@ -1765,5 +1765,58 @@ def test_is_leaf_node(self, dim, p, block_size, inplace):
         assert len(graph_node_names[0]) == 1 + op_obj.n_inputs
 
 
+class TestDiceLoss:
+    def get_reduction_method(self, reduction):
+        return {
+            "sum": torch.sum,
+            "mean": torch.mean,
+            "none": None
+        }[reduction]
+
+    @pytest.mark.parametrize("device", cpu_and_gpu())
+    def test_dice_loss(self, device):
+        input = torch.tensor([[[0.9409, 0.9220], [0.9524, 0.1094]],
+                              [[0.6802, 0.7949], [0.9570, 0.1499]],
+                              [[0.3298, 0.4401], [0.1094, 0.7536]],
+                              [[0.3340, 0.9895], [0.9563, 0.5045]]], device=device)
+        labels = torch.tensor([[[0, 1], [1, 0]],
+                               [[1, 0], [0, 1]],
+                               [[1, 0], [1, 0]],
+                               [[1, 0], [0, 1]]], device=device)
+        expected = torch.tensor([0.4028, 0.6101, 0.5916, 0.6347], device=device)
+        torch.testing.assert_allclose(ops.dice_loss(input, labels, eps=0), expected)
```

Comment:
```suggestion
        torch.testing.assert_allclose(ops.dice_loss(input_tensor, labels, eps=0), expected)
```

To avoid name clash with python input() method.